src/components/themed/SelectableRow.js

```diff
@@ -31,7 +36,7 @@ class SelectableRowComponent extends React.PureComponent<Props & ThemeProps> {
               <EdgeText style={styles.subTitle}>{subTitle}</EdgeText>
             </View>
           </View>
-          <IonIcon size={theme.rem(1.5)} color={this.props.theme.icon} name="chevron-forward-outline" style={styles.iconStyle} />
+          <IonIcon size={theme.rem(1.5)} color={arrowColor || theme.icon} name="chevron-forward-outline" style={styles.iconStyle} />
```

Comment:
Don't just pass in the 'arrowColor' as prop.
Instead, pass in a prop called 'tappable' and then ```color={tappable ? theme.iconTappable :  theme.icon}```

As a general rule, don't pass styles as props.